spec/templatingBehaviors.js

```diff
@@ -40,76 +69,92 @@ var dummyTemplateEngine = function (templates) {
         var templateOptions = options.templateOptions; // Have templateOptions in scope to support [js:templateOptions.foo] syntax
 
         var result;
-        with (bindingContext) {
```

Comment:
We've got to get `with` references out at some point.  Being generally an anti-pattern they tend to be confusing and also unoptimized in the browser.  We may need some alternative `lookup` here.

... if we need template rewriting at all ... what does it do? lol